table.sql

```diff
@@ -5,3 +5,5 @@ CREATE TABLE "session" (
 )
 WITH (OIDS=FALSE);
 ALTER TABLE "session" ADD CONSTRAINT "session_pkey" PRIMARY KEY ("sid") NOT DEFERRABLE INITIALLY IMMEDIATE;
+
+CREATE INDEX "IDX_session_expire" ON "session" ("expire") 
```

Comment:
Missing ; on the end of statement:

CREATE INDEX "IDX_session_expire" ON "session" ("expire");